scripts/database/pg/migrations/020.do.add-constraint-unique-eventbrite-event-id.sql

```diff
@@ -3,7 +3,7 @@ DO $$
     BEGIN
         ALTER TABLE cd_events ADD UNIQUE (eventbrite_id);
     EXCEPTION
-        WHEN duplicate_column THEN RAISE NOTICE 'constraint eventbrite_id already exists in cd_events.';
+        WHEN OTHERS THEN RAISE NOTICE 'constraint eventbrite_id already exists in cd_events.';
```

Comment:
The actual error message is... not very informative. 😅

Reply:
Well, if you look at 90% of the migration errors, they are useless and not even catching other scenario than duplicate columns..
Also, if i catch a migration error in prod, there will be blood.